components/Footer/utils.tsx

```diff
@@ -0,0 +1,26 @@
+import FacebookIcon from '@/icons/socialmedia/Facebook.svg'
+import InstagramIcon from '@/icons/socialmedia/Instagram.svg'
+import LinkedInIcon from '@/icons/socialmedia/LinkedIn.svg'
+import TwitterIcon from '@/icons/socialmedia/Twitter.svg'
+
+export const footerData = {
+  base: {
+    PlayMaker: 'https://xd.pl',
+    Regulamin: 'https://xd.pl',
+    'Warunki Prywatności': 'https://xd.pl',
+    'Dane firmy': 'https://xd.pl',
+    Kontakt: 'https://xd.pl',
+  },
+  services: {
+    Skauting: 'https://xd.pl',
+    'Wsparcie transferowe': 'https://xd.pl',
+    'Zwiększ limity': 'https://xd.pl',
+  },
+}
```

Comment:
mb if names are that similar do this and loop over object entries ? 
```suggestion
export const footerData = {
 Podstawowe: {
    PlayMaker: 'https://xd.pl',
    Regulamin: 'https://xd.pl',
    'Warunki Prywatności': 'https://xd.pl',
    'Dane firmy': 'https://xd.pl',
    Kontakt: 'https://xd.pl',
  },
  Usługi: {
    Skauting: 'https://xd.pl',
    'Wsparcie transferowe': 'https://xd.pl',
    'Zwiększ limity': 'https://xd.pl',
  },
}
```